selfhost/typechecker.jakt

```diff
@@ -1752,6 +1752,25 @@ struct Typechecker {
             }
             yield CheckedExpression::Garbage(span)
         }
+        ForcedUnwrap(expr, span) => {
+            let checked_expr = .typecheck_expression(expr, scope_id, safety_mode)
+            let optional_struct_id = .find_struct_in_prelude("Optional")
+            let UNKNOWN_TYPE_ID = TypeId(module: ModuleId(id: 0), id: 0)
+            let type_id = match .program.get_type(.expression_type(checked_expr)) {
+                GenericInstance(id, args) => {
+                    if not id.equals(optional_struct_id) {
```

Comment:
Oops I didn't see you also implemented `ForcedUnwrap` in #683. But you seem to missing the check for `WeakPtr`, was this a conscious decision or an oversight?

Reply:
Oversight. No worries! I'll rebase on yours.